### PR TITLE
TAJO-2021: Some refactoring and changing anonymous class to lambda expression

### PR DESCRIPTION
**Pull request by eminency**
It's some simplified code changes

### CODE REVIEW

File: tajo-core/src/main/java/org/apache/tajo/worker/TajoWorker.java [hunk 1]
```diff
@@ -242,27 +210,10 @@ private void initWorkerMetrics() {
...
-      public Integer getValue() {
-        if(taskExecutor != null) {
-          return taskExecutor.getRunningTasks();
-        } else {
-          return 0;
-        }
-      }
-    });
+    workerSystemMetrics.register(Node.QueryMaster.RUNNING_QM, () ->
+        queryMasterManagerService != null ? queryMasterManagerService.getQueryMaster().getQueryMasterTasks().size() : 0);
```
Comment (jinossy) [inline tajo-core/src/main/java/org/apache/tajo/worker/TajoWorker.java]: This line is over 120 char
https://cwiki.apache.org/confluence/display/TAJO/Code+Style

**Comment by jinossy** [comment]
Looks good to me but lambda expression does not support on jdk7
I’ve just change the target version to master

**Comment by eminency** [comment]
@jinossy Thanks for the review. I wrapped up the lines.

**Comment by jinossy** [comment]
+1 LGTM!
Thanks for your contribution

**Pull request closed without merging**